assignments/array-methods.js

```diff
@@ -56,21 +56,40 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
-console.log(fullName);
+    runners.forEach(function(object) {
+        let name=object.first_name+""+object.last_name;
+        fullName.push(name)
+    console.log(fullName);
+})
+
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+const bigLetters = (str) => {
+    return str.toUpperCase();
+}
+allCaps = fullName.map(bigLetters)
 console.log(allCaps); 
 
+
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
 let largeShirts = [];
+const sizeLarge = (object) => {
```

Comment:
```suggestion
let largeShirts = runners.filter(runner => runner.shirt_size === 'L'
```
So you can do an anonymous function and place it in the filter, you don't need to write a function first and then call it in the array method.